docs/development_suite/identity-and-access-management/service-account-management.md

```diff
@@ -0,0 +1,157 @@
+      We highly suggest to use this method whenever it is required not to share the credentials with the server or you cannot trust the network the Service Account is using.
+    ![Add Company Service Account with private key jwt auth](./img/service-account-management/add_company_sa_private_key_jwt.png)
+
+:::info
+Find out more about how these authentication methods work in the Console in the [Service Account authentication](/development_suite/identity-and-access-management/service-account-management.md#service-account-authentication) paragraph.
+:::
+### Editing a Service Account Role
+
+A Service Account Role in the Company can be modified: to do so, simply click on the edit button for the desired Service Account row and select the new Role.
+
+![Edit Company Service Account](./img/service-account-management/edit_company_sa.png)
+
+### Removing a Service Account from the Company
+
+A Service Account can be removed from the Company by clicking the delete icon on the table and confirming the action.
+
+![Delete Company Service Account](./img/service-account-management/delete_company_sa.png)
+
+:::warning
+Removing a Service Account from the Company will permanently delete the account and all its existing Roles, which will be lost and cannot be recovered.
+:::
+
+## Managing Project and Environment Service Accounts
+
+A User with enough administrative permission on a specific Project will be able to view all the existing Service Accounts in the Company and edit their Role on the specific Project (and, optionally, on each existing Runtime Environment).
+
+![Project identities page](./img/service-account-management/project_identities.png)
+
+:::note
+Although the Project Identities administration portal shows all the Company's identities, this does not mean that all of the identities have access to the Project, since this depends on the Role they are assigned in the Company and how the permissions are inherited.
+
+For further information about permissions and Role inheritance check out the [Console Levels and Permission Management](/development_suite/identity-and-access-management/console-levels-and-permission-management.md) page.
+:::
+
+### Adding a new Service Account at Project level
+
+The Company Owner can add a new Service Account by clicking on the *Add Service Account* option inside the *Add User* dropdown, on the top-right corner of the Identities section of the Project settings area. Here, the Service Account can be assigned a Company Role and additional Roles on the specific Project and its Runtime Environments. 
+
+The Service Account invitation process will also require an authentication method, as specified in the [Adding a new Service Account](/development_suite/identity-and-access-management/service-account-management.md#adding-a-new-service-account) paragraph.
```

Comment:
```suggestion
The Service Account invitation process will also require an authentication method, as specified in the [Adding a new Service Account](/development_suite/identity-and-access-management/service-account-management.md#adding-a-new-service-account) paragraph.

:::note
Note that new Service Accounts are always instantiated at Company level: you can create a Service Account from the Project Settings page, but it will still be shared with the whole Company.
:::
```